pkg/server/server.go

```diff
@@ -232,6 +233,17 @@ func coreControllers(ctx context.Context, sc *Context, config *Config) error {
 		return err
 	}
 
+	if config.ControlConfig.EncryptSecrets {
+		if err := secretsencrypt.Register(ctx,
+			sc.K8s,
+			&config.ControlConfig,
+			sc.Core.Core().V1().Node(),
+			sc.Core.Core().V1().Secret(),
+			sc.K8s.CoreV1().Events("")); err != nil {
```

Comment:
You shouldn't need to pass this in separately, we're already passing in `sc.K8s` as `kubernetes` and you can call `kubernetes.CoreV1().Events("")` from inside `secretsencrypt.Register`. If I remember correctly, we pass in the other core controllers individually because we can't just pass in sc.Core without creating an import cycle.

Reply:
Consolidated into single util function.